ansible_collections/arista/avd/roles/eos_designs/templates/base/snmp-server.j2

```diff
@@ -18,4 +23,40 @@ snmp_server:
 {%         set snmp_location = snmp_location ~ " " ~ inventory_hostname %}
     location: {{ snmp_location }}
 {%     endif %}
+{%     if snmp_settings.users is arista.avd.defined %}
+    users:
+{%         for user in snmp_settings.users %}
+    - name: {{ user.name }}
+      group: {{ user.group }}
+      version: {{ user.version }}
+{%             if user.version is arista.avd.defined("v3")
+                 and snmp_settings.auto_compute_local_engineid is arista.avd.defined(true)
+                 and snmp_settings.compute_v3_user_localized_key is arista.avd.defined(true) %}
+      localized:  {{ local_engine_id }}
+{%             endif %}
+{%             if user.version is arista.avd.defined("v3")
+                and user.auth is arista.avd.defined
+                and user.auth_passphrase is arista.avd.defined %}
+      auth: {{ user.auth }}
+{%                 if snmp_settings.auto_compute_local_engineid is arista.avd.defined(true)
+                      and snmp_settings.compute_v3_user_localized_key is arista.avd.defined(true) %}
+{%                     set hash_filter = {"passphrase": user.auth_passphrase, "auth": user.auth, "engine_id": local_engine_id} %}
+      auth_passphrase: {{ hash_filter | hash_passphrase }}
```

Comment:
please use FQCN ex. `arista.avd.hash_passphrase`. Same elsewhere. Also make sure if you are adding new requirements.

Reply:
I am not sure I understand this part 
> Also make sure if you are adding new requirements. 

What do you mean?

Reply:
so `hash_passphrase` is the new filter and `hash` is the ansible builtin one nothing else present. Once I figure out the FQCN for the builtin filter I guess we are good for this one?